waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/util/MetaStoreUriNormaliser.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.waggledance.util;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Set;
+import java.util.TreeSet;
+
+import com.google.common.base.Joiner;
+
+public class MetaStoreUriNormaliser {
```

Comment:
Similar, why is this new class being introduced?

Reply:
I see am I not the only confused. I did some digging seems this was a static method before.
It is just hard to follow when stuff is moving and refactored in the same PR. @courtsVII perhaps two PR's one after the other are a better idea. One for moving stuff as is, one for doing the refactoring. Makes it a lot easier to review/trace back later etc...

Reply:
I'm closing this PR, it doesn't need attention, I am just keeping this branch up to date so I don't have to do a massive context switch and make design decisions I regret when I have to change the project to use new library